src/git-source-provider.ts

```diff
@@ -0,0 +1,285 @@
+import * as core from '@actions/core';
+import * as coreCommand from '@actions/core/lib/command';
+import * as fs from 'fs';
+import * as fsHelper from './fs-helper';
+import * as gitCommandManager from './git-command-manager';
```

Comment:
I would import the individual classes/interfaces if you are using 2 or less (in this case you are using the type a lot so it would make it simpler to declare variables)

Reply:
+1 i ended up using this pattern for classes/interfaces, not for exported functions though.